src/lib/style/components/_toasts.scss

```diff
@@ -0,0 +1,21 @@
+.SynapseToastContainer {
+  inset: unset !important;
```

Comment:
I was pretty heavy-handed with the `!important`s here but figured it's not a huge deal since it's only affecting our toast messages. Let me know if you think this is a problem and I can spend some more time trying to work around it.